spec/spec_helper.rb

```diff
@@ -9,6 +20,10 @@
 SimpleCov.start
 
 RSpec.configure do |config|
+  config.before(:each) do
+    # test database
+  end
+
```

Comment:
I like your prep work here. Its good that you're making things easier by setting up the code for the test environment here.